triage/triage-team.rst

```diff
@@ -46,9 +42,9 @@ following:
 If a triager has any doubt about whether to close a PR, they should consult a core
 developer before taking any action.
 
-Triagers can also make use of the ``invalid`` and ``stale`` labels to suggest that a
+Triagers can also make use of the ``pending`` and ``stale`` labels to suggest that a
 PR may be suitable for closure. For more information, see the
-:ref:`GitHub PR labels <github-pr-labels>` section.
+:ref:`GitHub PR labels <gh-labels>` section.
```

Comment:
```suggestion
entries in the :ref:`Other` section.
```

* Link to the specific section the labels are in
* Mention referring to the specific entries there, rather than the general section
* If we're referring to the section, use the actual section title rather than a manual hardcoded one

Reply:
This title is not particularly informative though.

Reply:
In that case, then, we should either rename the section (to `Other Labels`), provide the necessary context in the prose around the link:

```suggestion
entries in the :ref:`Other` label section.
```

or add ref targets right above each of the labels' respective entries in the list, and link directly to them (with custom link text):

```suggestion
:ref:`pending <label-pending>` and :ref:`stale <label-stale>` entries in the :ref:`Other` section.
```